civis/io/_files.py

```diff
@@ -198,10 +200,17 @@ def file_to_civis(buf, name, api_key=None, client=None, **kwargs):
     file_id : int
         The new Civis file ID.
 
+    Raises
+    ------
+    ValueError
```

Comment:
I think this should be a `TypeError` -- it looks like you get it when the type of the input is incorrect (i.e., a file-like object with no `name`), yes?
```suggestion
    TypeError
```

Reply:
Agreed that `TypeError` makes more sense here, as it's more like a "missing an argument" error. Code updated at c6fc7a3.